db/org_members_groups.go

```diff
@@ -368,6 +368,8 @@ func (ms *MongoStorage) CheckGroupMembersFields(
 			// and continue to the next member
 			results.Duplicates = append(results.Duplicates, m.ID)
 			results.Duplicates = append(results.Duplicates, val)
```

Comment:
since this will be deduped, it might be more effective to just use a map in the first place, and then convert to a slice at the end

Reply:
i propose this in #197